src/queue.js

```diff
@@ -1,5 +1,21 @@
 class Queue {
-
+  constructor() {
+    this.queue = [];
+  }
+  enqueue(...args) {
+    this.queue.push(...args);
+  }
+  dequeue(...args) {
```

Comment:
Passing `...args` to enqueue makes sense so that you can pass in multiple inputs at once, but it doesn't make a lot of sense passing it to dequeue, since you don't pass any inputs to it when you call it. It just dequeues the next element.